homeassistant/components/hunterdouglas_powerview/number.py

```diff
@@ -0,0 +1,120 @@
+"""Support for hunterdouglass_powerview sensors."""
```

Comment:
We shouldn't add a new platform at the same time. This should come in a future PR

Reply:
this is needed to support 'velocity' which is new in Gen3 hubs